Plugins/BCClient/Source/BCClientPlugin/Private/BrainCloudClient.cpp

```diff
@@ -1,8 +1,8 @@
 // Copyright 2018 bitHeads, Inc. All Rights Reserved.
 
-#include "BCClientPluginPrivatePCH.h"
-#include "BrainCloudClient.h"
 
+#include "BrainCloudClient.h"
+#include "BCClientPluginPrivatePCH.h"
```

Comment:
ya ok. so in the past, the PCH not being first would cause issues.... and that one make sense, because Precompiled Headers should always be put first. But now they changed the rule?